main/integration-tests/__test__/api-tests/user/get-user.test.js

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Get current user scenarios', () => {
+  let setup;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting current user', () => {
+    it('should fail for anonymous user', async () => {
+      const anonymousSession = await setup.createAnonymousSession();
+      await expect(anonymousSession.resources.currentUser.get()).rejects.toMatchObject({
+        code: errorCode.http.code.badImplementation,
```

Comment:
It is interesting that the API is returning this response, maybe we can add a comment to the first tab in the spreadsheet to see if it possible to return a more appropriate response code

Reply:
+1 I'll add it to the spreadsheet 

Sending a wrong/expired token does get an Unauthorized error, but when the token is missing a 500 is thrown